Sources/StreamChat/APIClient/APIClient_Tests.swift

```diff
@@ -4,6 +4,8 @@
 
 @testable import StreamChat
 import XCTest
+@testable
+import StreamChatTestTools
```

Comment:
Are you sure you have the formatter config properly in the pre-commit hook? 🤔

Reply:
It does trigger every time I commit 🤔 
```
Running SwiftFormat...
Swiftformat completed successfully.
```